tasks/views.py

```diff
@@ -1,3 +1,62 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.urls import reverse_lazy
+from django.views import generic
+
+from .models import Task, Tags

```

Comment:
Using absolute imports is considered good practice and it helps make your code more readable and maintainable.

Reply:
not fixed